web/src/server_message.ts

```diff
@@ -55,7 +55,6 @@ export const server_message_schema = z.intersection(
         sender_email: z.string(),
         sender_full_name: z.string(),
         sender_id: z.number(),
-        sender_realm_str: z.string(),
```

Comment:
So I think we need to be careful with changes like this. If the client isn't using it, but the server includes it in the API, we should leave a comment here noting that we intentionally don't parse it since it is unused by the web app. You can see some good examples of this pattern in `zulip-flutter`, I expect.